nums/models/glms.py

```diff
@@ -286,7 +286,8 @@ def hessian(self, X: BlockArray, y: BlockArray, mu: BlockArray = None):
         if self._penalty is None:
             return X.T @ (s * X)
```

Comment:
On page 7 [here](https://www.cs.ubc.ca/~nando/540-2013/lectures/l11.pdf) , hessian is X.T @ (diag(s) * S). Am i missing something?